mapper/orm/src/main/java/org/hibernate/search/mapper/orm/session/impl/HibernateOrmSearchSessionSession.java

```diff
@@ -50,23 +50,24 @@
 /**
  * The actual implementation of {@link SearchSession}.
  */
-public class HibernateOrmSearchSession extends AbstractPojoSearchSession
-		implements SearchSession, HibernateOrmScopeSessionContext, ReferenceHitMapper<EntityReference> {
+public class HibernateOrmSearchSessionSession extends AbstractPojoSearchSession
```

Comment:
I know that name will be reverted by a subsequent commit. But is there a reason to have it in the history?

Reply:
I squashed a fix-up commit into the wrong commit. Fixed, thanks.